### PR TITLE
Update to Geozero 0.7

### PR DESCRIPTION
**Pull request by pka**
geozero has a major update which combines geozero-core and geozero into one crate and has new highlevel traits like `ToJson`.

### CODE REVIEW

**Comment by pka** [comment]
@michaelkirk Would be great, if you could review the PR. Would like to make release after this.

**Review by michaelkirk — APPROVED** [review]
This all LGTM! 

My only feedback is minor, feel free to ignore it.

If you're curious, here's what updating abstreet to use this would look like: 
https://github.com/michaelkirk/abstreet/commit/ec5ce7b40e48d33a75665765819d32908a34833c

File: src/rust/src/lib.rs [hunk 1]
```diff
@@ -23,8 +17,8 @@
 //! let mut fgb = FgbReader::open(&mut filein)?;
 //! fgb.select_bbox(8.8, 47.2, 9.5, 55.3)?;
 //! while let Some(feature) = fgb.next()? {
-//!     let props = feature.properties()?;
-//!     println!("{}", props["name"]);
+//!     println!("{}", feature.property("name").unwrap_or("?".to_string()));
```
[the remark this answers is not in the document]
Comment (pka) [reply src/rust/src/lib.rs:20]: In this case `unwrap_or` defines the target type of the `property` method...

**Comment by pka** [comment]
The change in abstreet looks like I hoped!

You could replace
```
let props = feature.properties()?;
let population: usize = props["population"].parse()?;
```
with e.g.
```
let population: Option<usize> = feature.property("population");
```
to avoid a string conversion.

File: src/rust/src/lib.rs (hunk 1 above)
[the remark this answers is not in the document]
Comment (michaelkirk) [reply src/rust/src/lib.rs]: Ah, thanks for explaining.

I like the new version with the explicit turbofish better!

**Comment by michaelkirk** [comment]
> let population: Option<usize> = feature.property("population");

Excellent! I knew there had to be a better way, Thank you!

**Review by bjornharrtell — APPROVED** [review]

**Pull request merged**